### PR TITLE
Ticket #27925 Wrap lines at 79 chars

### PR DESCRIPTION
**Pull request by za**
Initial PR is here:
https://github.com/django/django/pull/14200

credit goes to https://github.com/kshitijraghav

### CODE REVIEW

File: docs/howto/deployment/wsgi/modwsgi.txt [hunk 1]
```diff
@@ -97,44 +97,96 @@ should put in this file, and what else you can add to it.
...
+
+For further information related to using mod_wsgi-express see the main mod_wsgi documentation.
 
 .. _serving-files:
 
 Serving files
 =============
 
-Django doesn't serve files itself; it leaves that job to whichever web
+Django doesn't serve files itself; it leaves that job to whichever Web
```
Comment (felixxm) [inline docs/howto/deployment/wsgi/modwsgi.txt:186]: `Web` should not be capitalized, see [docs](https://docs.djangoproject.com/en/4.0/internals/contributing/writing-documentation/#commonly-used-terms).

Comment (felixxm) [reply docs/howto/deployment/wsgi/modwsgi.txt]: Not only in this line. Please revert changes to other lines where `web` is switched to `Web`.

Comment (za) [reply docs/howto/deployment/wsgi/modwsgi.txt]: Aah, OK @felixxm Fixed with this commit https://github.com/django/django/pull/14939/commits/8034353caad11f06364b1a08181b7bd9d9b4a9ca

**Review by blockchainGuru1018 — APPROVED** [review]

File: docs/howto/deployment/wsgi/modwsgi.txt [hunk 2]
```diff
@@ -97,34 +97,86 @@ should put in this file, and what else you can add to it.
...
-(``https://example.com/mysite`` in this example), you can add ``WSGIScriptAlias``
-to the configuration above:
+This will compile ``mod_wsgi`` and install the resulting package into your
+Python installation. If wishing to install an official release direct from
+the Python Package Index ``(PyPi)``, you can instead run:
 
 .. code-block:: apache
 
-    WSGIScriptAlias /mysite /path/to/mysite.com/mysite/wsgi.py process-group=example.com
+    pip install mod_wsgi
```
Comment (jacobtylerwalls) [inline docs/howto/deployment/wsgi/modwsgi.txt:116]: Django style is to make this more verbose to help beginners avoid mistakes, see ticket-30367
```suggestion
    python -m pip install mod_wsgi
```

File: docs/howto/deployment/wsgi/modwsgi.txt [hunk 3]
```diff
@@ -97,34 +97,86 @@ should put in this file, and what else you can add to it.
...
+standard location, you can set and export the APXS environment variable to the
+location of the Apache APXS script for your Apache installation before
+performing the installation. If you are using ``Linux``, ``macOS`` or other
+``UNIX`` type operating system, and you don't have Apache available, you can
+instead install ``mod_wsgi`` using:
 
-.. _details on setting up daemon mode: https://modwsgi.readthedocs.io/en/develop/user-guides/quick-configuration-guide.html#delegation-to-daemon-process
+.. code-block:: apache
+
+    pip install mod_wsgi-standalone
```
Comment (jacobtylerwalls) [inline docs/howto/deployment/wsgi/modwsgi.txt:127]:
```suggestion
    python -m pip install mod_wsgi-standalone
```

File: docs/howto/deployment/wsgi/modwsgi.txt (hunk 2 above)
Comment (jacobtylerwalls) [reply docs/howto/deployment/wsgi/modwsgi.txt:116]: Greetings. Still needs updating.

**Comment by felixxm** [comment]
@za Thanks for this patch :+1:, however it's almost entirely copied from `mod_wsgi` [README](https://github.com/GrahamDumpleton/mod_wsgi#installation-into-python). We would like to add a minimal working example, not copy `mod_wsgi` docs with all details (see [comment:3](https://code.djangoproject.com/ticket/27925#comment:3) and [comment:5](https://code.djangoproject.com/ticket/27925#comment:5)).

**Comment by felixxm** [comment]
Closing due to inactivity.

**Pull request closed without merging**